x-pack/plugins/canvas/canvas_plugin_src/renderers/pie/index.tsx

```diff
@@ -22,9 +22,6 @@ export const pie: RendererFactory<Pie> = () => ({
   help: strings.getHelpDescription(),
   reuseDomNode: false,
   render: async (domNode, config, handlers) => {
-    // @ts-expect-error
-    await import('../../lib/flot-charts');
-
     if (!includes($.plot.plugins, piePlugin)) {
```

Comment:
It seems the pie plugin already exists in `ui-shared-deps`.
Could you please check if it is removable?
At first sight it seems yes.

Reply:
I checked this. We have two different objects. 'Includes' from lodash in this case returns false when compare them and we always put another pie plugin with the same name. I think we should leave the second version because we put after the first 'pie' plugin because of this I think we use the second version pf pie in canvas.

Reply:
I see some minor diff in 2 versions of `pie` plugin , but both files use v1.1  Agree with @sulemanof if we move something into `ui-shared-deps` we should use that from that place

![image](https://user-images.githubusercontent.com/20072247/95751014-a2c73200-0ca6-11eb-89b1-3724d1664a6d.png)